tests/user/test_users.yml

```diff
@@ -54,7 +54,7 @@
       users:
       - name: user1
         givenname: user1
-        last: Last
+        sn: Last
```

Comment:
Only change one for the test please. This way the test will make sure that `last` and `sn` are the same.